Platform/TigerlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -457,6 +457,7 @@ UpdateFspConfig (
   if (SiCfgData != NULL) {
     Fspmcfg->PchHdaEnable = SiCfgData->PchHdaEnable;
     Fspmcfg->PchHdaDspEnable = SiCfgData->PchHdaDspEnable;
+    CopyMem (&Fspmcfg->PchHdaAudioLinkDmicEnable, SiCfgData->PchHdaAudioLinkDmicEnable, sizeof(SiCfgData->PchHdaAudioLinkDmicEnable));

```

Comment:
Is there a specific reason to CopyMem instead of atomically assigning? It is one byte long.